### PR TITLE
SD Patches, Software, SP migration

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

modules/reference/pages/systems/system-details/sd-patches.adoc
modules/reference/pages/systems/system-details/sd-software.adoc
modules/reference/pages/systems/system-details/sd-sp-migration.adoc

# Target branches

To what product version this document applies?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Partial https://github.com/SUSE/spacewalk/issues/11449

### CODE REVIEW

File: modules/reference/pages/systems/system-details/sd-software.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [[sd-software]]
-= SD Software
+= Software
 
-This tab and its subtabs allow you to manage the software on the system: patches (errata), packages and package profiles, software channel memberships, and migrations.
\ No newline at end of file
+The menu:Systems[Software] section allows you to manage software on the selected client.
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-software.adoc:4]:
```suggestion
The menu:System Details[Software] section allows you to manage software on the selected client.
```
Maybe, we'd better say: "The Software section of the System Details page ..." (I'm not sure about the adoc decoration, though).

File: modules/reference/pages/systems/system-details/sd-sp-migration.adoc [hunk 2]
```diff
@@ -1,8 +1,14 @@
 [[sd-sp-migration]]
-= SD Service Pack Migration
+= SP Migration
 
-Service Pack Migration (SP Migration) allows you to upgrade a system from one service pack to another.
+The menu:Systems[Software > SP Migration] section allows you to upgrade clients to the latest service pack of their underlying operating system.
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-sp-migration.adoc:4]: This is also "System Details" page.

File: modules/reference/pages/systems/system-details/sd-sp-migration.adoc [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+Check the product version you want to upgrade the client to, and click btn:[Select Channels]
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-sp-migration.adoc:5]:
```suggestion
Check the product version you want to upgrade the client to, and click btn:[Select Channels].
```

File: modules/reference/pages/systems/system-details/sd-patches.adoc [hunk 4]
```diff
@@ -1,28 +1,10 @@
 [[sd-patches]]
-= SD Patches
+= Patches
 
-This subtab contains a list of patch (errata) alerts applicable to the system.
-For the meanings of the icons used in this tab, see xref:installation:webui-setup.adoc[].
+The menu:Systems[Software > Patches] section displays a list of all patches currently available to the selected client.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/reference/pages/systems/system-details/sd-patches.adoc:4]: image

**Pull request merged**